invokeai/frontend/web/src/services/api/schema.d.ts

```diff
@@ -5252,17 +5259,17 @@ export type components = {
       image?: components["schemas"]["ImageField"];
     };
     /**
-     * StableDiffusion2ModelFormat 
+     * StableDiffusionXLModelFormat 
```

Comment:
Is this intentional?

Reply:
This file is autogenerated based on our back-end route schema